src/main/webapp/view.html

```diff
@@ -11,6 +11,7 @@
   crossorigin="anonymous"
 />
 <script src="/discussion.js" defer></script>
+<script src="/transcript.js" defer></script>
```

Comment:
There is already a `transcript.js` in this html file. Can you remove one of them?